acceptCard.py

```diff
@@ -74,3 +74,78 @@ async def acceptCard(
     if newCard:
         cardSheetUnapproved.update_cell(dbRowIndex, 1, cardName)
         cardSheetUnapproved.update_cell(dbRowIndex, 3, authorName)
+
+async def acceptVetoCard(
+        bot: commands.Bot,
+        cardMessage: str,
+        file: discord.File,
+        cardName: str,
+        authorName: str
+):
+    extension = re.search("\.([^.]*)$", file.filename)
+    fileType = (
+        extension.group() if extension else ".png"
+    )  # just guess that the file is a png
+    new_file_name = f'{cardName.replace("/", "|")}{fileType}'
+    image_path = f"tempImages/{new_file_name}"
+
+    file_data = file.fp.read()
+    file_copy_for_cardlist = discord.File(
+        fp=io.BytesIO(file_data), filename=new_file_name
+    )
+    cardListChannel = cast(
+        discord.TextChannel, bot.get_channel(hc_constants.SIX_ONE_CARD_LIST)
+    )
+    vetoCardListChannel = cast(
+        discord.TextChannel, bot.get_channel(hc_constants.VETO_CARD_LIST)
+    )
+
+
+    async for message in cardListChannel.history(limit=None):
+        if message.content == cardMessage:
+            try:
+                await message.delete()  # Delete message if it matches
+                print(f"Deleted message: {message.content}")
+            except discord.Forbidden:
+                print("Bot lacks permissions to delete messages.")
+            except discord.HTTPException as e:
+                print(f"Failed to delete message: {e}")
+
+    async for message in vetoCardListChannel.history(limit=None):
```

Comment:
Why does it delete it from the veto card list?

Reply:
I assume just so that there aren't duplicates of old versions in #card-list-corrrupted

Reply:
I was just following Liu's instructions